src/fetchers/WelcomePage/WelcomPage.tsx

```diff
@@ -22,7 +23,9 @@ const WelcomePage = () => {
                         buttonText="continue"
                         icon={true}
                         fullWidth={true}
-                        onClick={onClickHandler}
+                        onClick={() => loginWithRedirect({
+                            redirectUri: 'http://localhost:3000/home' 
```

Comment:
Not critical, but I believe it would have been better to put redirect Uri in a const outside of this component and import it